pandas/io/tests/parser/common.py

```diff
@@ -1467,5 +1469,23 @@ def test_memory_map(self):
             'c': ['I', 'II', 'III']
         })
 
-        out = self.read_csv(mmap_file, memory_map=True)
-        tm.assert_frame_equal(out, expected)
+        result = self.read_csv(mmap_file, memory_map=True)
+        tm.assert_frame_equal(result, expected)
+
+    def test_read_csv_utf_aliases(self):
+        # see gh issue 13549
+        path = 'test.csv'
+        expected = DataFrame({'A': [0, 1], 'B': [2, 3],
```

Comment:
I suppose we could do one row as 

expected = pd.DataFrame({'mb_num': [4.8], 'multibyte': ['test']})

I used BytesIO because I don't think StringIO can support different encodings (I tried and wasn't able to get StringIO to work).